target/linux/ramips/dts/mt7621_asus_4g-ax56.dts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "asus,4g-ax56", "mediatek,mt7621-soc";
+	model = "ASUS 4G-AX56";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+		bootargs-override = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: led-0 {
+			color = <LED_COLOR_ID_WHITE>;
```

Comment:
please add label to power LEDs, else it doesn't work properly, consider this in future PRs too, thanks.
`label = "white:power";`
Also why `led-0` ? name it `power` or something.